src/entrypoints/my-redirect.ts

```diff
@@ -108,7 +110,12 @@ const render = (showTroubleshooting: boolean) => {
 
   let changeInstanceEl = document.querySelector(".instance-footer")!;
   changeInstanceEl.innerHTML = `
-    <b>Your instance URL:</b> ${instanceUrl}
+    <b>Your instance URL:</b>
+    ${instanceUrls.length === 1 
+      ? instanceUrl
+      : `<select id="instance-selector">
```

Comment:
Current:
![image](https://user-images.githubusercontent.com/1444314/195940182-dfb29638-ad89-4b40-9c89-749a2dfd85c7.png)

Instead of a dropdown, can we change the "Open Link" to be list items like this:

![image](https://user-images.githubusercontent.com/1444314/195940768-236b8de8-577e-4efc-a378-d8de908f9d9e.png)

Reply:
(example code https://github.com/esphome/dashboard/blob/main/src/install-choose/install-choose-dialog.ts#L53)